app/views/users/index.html.erb

```diff
@@ -1 +1,11 @@
-<h1>Users List</h1>
\ No newline at end of file
+<h1>List of users:</h1>
+<% @users.each do |user| %>
+  <hr/>
+  <ul>
+    <li><h2>Name: <%= user.name %></h2></li>
+    <li><img src ='<% user.photo %>' alt='user_image'  /></li>
+    <li>Number of posts:<%= user.posts_counter %></li>
+    <li><a href='/users/<%= user.id %>'> More info </a> </li>
+  </ul>
+  <br/> 
+<% end %>
```

Comment:
- [ ] Please make the user image appear on the left side and the user data on the right side and arrange the placement of all elements to make it match the given design.

- [ ] [Optional][Highly Recommended] Kindly add the user data here as a partial so you can use it in other views instead of repeating your code. See [this video](https://www.youtube.com/watch?v=c1VZ_0taJgQ) for more info.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/198069124-128cb960-bf77-4820-8efe-32180dfa0920.png)  | ![image](https://user-images.githubusercontent.com/68796232/198069152-9f4f8dbe-6959-4194-ab61-506bdb297902.png) |

Reply:
@Omar-Muhamad Thank you for reviewing my project, I will make the necessary corrections as suggested